examples/dash_with_statebuilder.py

```diff
@@ -9,37 +9,47 @@
 import dash
 import pandas as pd
 from dashdataframe import configure_app
-from neuroglancer_annotation_ui.statebuilder import StateBuilder
+from neuroglancer_annotation_ui import statebuilder
 
 FLASK_PORT = 8880
 
 df_filename = 'dash_example_data.h5'
 df_key = 'data'
-plot_columns = ['num_pre', 'num_post']
+plot_columns = ['pre_syn', 'post_syn']
+
+
+def _statebuilder():
+    image_layer = statebuilder.ImageLayerConfig(name='EM',
+                                                source='precomputed://gs://neuroglancer/pinky100_v0/son_of_alignment_v15_rechunked')
+    seg_layer = statebuilder.SegmentationLayerConfig(name='layer23',
+                                                     source='precomputed://gs://microns_public_datasets/pinky100_v185/seg',
+                                                     selected_ids_column='pt_root_id')
+    points = statebuilder.PointMapper('pt_position')
+    anno_layer = statebuilder.AnnotationLayerConfig(
+        name='annos', mapping_rules=points)
+    return statebuilder.StateBuilder([image_layer, seg_layer, anno_layer])
+
 
 def build_dash_app():
     df = pd.read_hdf(df_filename, df_key)
 
     app = dash.Dash()
 
-    sb = StateBuilder(dataset_name='pinky100',
-                      selected_ids={'seg': ['pt_root_id']},
-                      point_annotations={'soma': ['pt_position']})
-    
+    sb = _statebuilder()
```

Comment:
isnt' this _ a typo?

Reply:
Not a typo, but less clear than it should be. Changed.